test/unit/spec/function_output/KBaseSets/types/AssemblySet-spec.js

```diff
@@ -0,0 +1,33 @@
+define([
+    'react',
+    'react-dom',
+    'test/unit/utils/domUtils',
+    'widgets/function_output/KBaseSets/types/AssemblySet',
+    'json!../data/assemblyset_1.json'
+], (React, ReactDOM, domUtils, AssemblySet, testData1) => {
+    'use strict';
+
+    const { createElement } = React;
+    const { render } = ReactDOM;
+
+    describe('The Loader class', () => {
+        it('should be defined', () => {
+            expect(AssemblySet).toBeDefined();
+        });
+
+        it('should render a valid assembly set', async () => {
+            const node = document.createElement('div');
+
+            // install our component there
+            const el = createElement(AssemblySet, { set: testData1, setItem: () => { } }, null);
+            render(el, node);
+
+            await domUtils.waitFor(() => {
+                return node.innerText.includes(testData1.value.description);
+            }, 1000);
+
+            expect(true).toBeTrue();
```

Comment:
Yes, yes. Entertaining to be sure. I was building out those tests and this keeps the test runner from complaining. The exceptions thrown above it would fail the test, but the test runner fails without an official expectation in the test, so the final always-true expectation ensures that the test succeeds if there are no exceptions thrown above it.
Like I said, I was building out these tests bit by bit, finding patterns.